protocols/indexer/contracts/Indexer.sol

```diff
@@ -83,7 +94,7 @@ contract Indexer is IIndexer, Ownable {
     */
   function addToBlacklist(
     address _token
-  ) external onlyOwner {
+  ) external onlyOwner whenNotPaused {
```

Comment:
Given that the owner can pause and unpause, we can do without the restriction here and leave to `onlyOwner`. Same for `removeFromBlacklist`

Reply:
I am inclined to leave the double restriction on `killContract` to help prevent any accidental calls etc. It seems like a function worth protecting